var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -194,7 +194,8 @@ class Acts(CMakePackage, CudaPackage):
     version("0.08.1", commit="289bdcc320f0b3ff1d792e29e462ec2d3ea15df6")
     version("0.08.0", commit="99eedb38f305e3a1cd99d9b4473241b7cd641fa9")
 
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
```

Comment:
Can we narrow this? Is this enough then, or are there other dependencies that similarly require C? I was looking earlier into the DD4hep CMake config.cmake.in template, but couldn't identify where the C dependency comes from exactly.
```suggestion
    depends_on("c", type="build", when="+dd4hep")  # DD4hep requires C
```

Reply:
I want to check that too. We have already logic to inject dependencies, but I'm not keen of abusing it for these kind of dependency injection (`dd4hep` injecting dependency on C), since it might result in a lot of more rules for clingo. Currently this logic is used to inject runtimes (e.g. `gcc-runtime` from `gcc`).